src/java/htsjdk/variant/variantcontext/Genotype.java

```diff
@@ -181,6 +181,10 @@ public boolean hasDP() {
     // ---------------------------------------------------------------------------------------------------------
 
     /**
+     * if {@link #getAlleles()} is empty, the type is {@link GenotypeType#UNAVAILABLE}.
+     * if all the  alleles are NoCall,  the type is {@link GenotypeType#NO_CALL}.
+     * if some, but not all, alleles are NoCall,  the type is {@link GenotypeType#MIXED}.
+     *
```

Comment:
Your doc for this method is missing any description of what happens when **none** of the alleles are no call (ie., you get `GenotypeType.HET`, `GenotypeType.HOM_REF`, or `GenotypeType.HOM_VAR` as appropriate).